.changeset/rotten-geese-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`AdvancedTable` - Added `@maxHeight` argument which automatically adds a sticky header when set and sets the max height of the AdvancedTable. Also updated the container styles to constrain the Advanced Table width to the parent's width.
```

Comment:
I would put the added features on separate lines as done here: https://helios.hashicorp.design/components/code-editor?tab=version%20history

Added `@maxHeight` argument which automatically adds a sticky header when set and sets the max height of the `AdvancedTable`.

Updated the container styles to constrain the `AdvancedTable` width to the parent's width.

Reply:
I think were not really supposed to do the multiple lines anymore.. the automation doesn't know to look for line breaks